src/mbgl/shader/circle_uniforms.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <mbgl/shader/uniforms.hpp>
+
+namespace mbgl {
+
+MBGL_DEFINE_UNIFORM_SCALAR(float, u_radius);
```

Comment:
Ok -- there aren't any cases of that currently, and I think we should avoid introducing any. Besides link errors, it would just be confusing to use the same uniform name for different types.